### PR TITLE
VoiceControl Plugin json and md files update

### PR DESCRIPTION
**Pull request by sivapriya201**
Documentation Enhancement of VoiceControl Plugin
jira : https://ccp.sys.comcast.net/browse/RDK-36558

### CODE REVIEW

File: VoiceControl/VoiceControl.json [hunk 1]
```diff
@@ -135,23 +135,35 @@
             }
         },
         "sendVoiceMessage": {
-            "summary": "Sends a message to the Voice Server. The specification of this message is not in the scope of this document. Example use cases for this API call include sending context or sending ASR blobs to the server.",
+            "summary": "Sends a message to the Voice Server. The specification of this message is not in the scope of this document. Example use cases for this API call include sending context or sending ASR blobs to the server.\n\nEvents\n\n| Event | Description |\n| :-------- | :-------- |\n| `onSessionBegin` |Triggers if the voice session begins |\n| `onStreamBegin` |Triggers if a device starts streaming voice data to the RDK|\n| `onKeywordVerification` |Triggers if a keyword verification result is received.|\n| `onServerMessage` |Triggers if a message is received from the Voice Server |\n| `onStreamEnd` |Triggers if streaming audio is stopped from the device |\n| `onSessionEnd` |Trigger if interaction with the server is end|",
+            "events":[
```
Comment (cbucht200) [inline VoiceControl/VoiceControl.json:139]: These events are not related to sendVoiceMessage

File: VoiceControl/VoiceControl.json [hunk 2]
```diff
@@ -455,7 +530,8 @@
                 "required": [
                     "remoteId",
                     "sessionId",
-                    "reason"
+                    "reason",
+                    "serverstats"
```
Comment (cbucht200) [inline VoiceControl/VoiceControl.json:534]: I believe serverstats is optional

File: docs/api/VoiceControlPlugin.md [hunk 3]
```diff
@@ -166,20 +170,33 @@ Configures the RDK's voice stack. NOTE: The URL Scheme determines which VREX API
...
 ```
 
-<a name="sendVoiceMessage"></a>
-## *sendVoiceMessage*
+<a name="method.sendVoiceMessage"></a>
+## *sendVoiceMessage [<sup>method</sup>](#head.Methods)*
 
 Sends a message to the Voice Server. The specification of this message is not in the scope of this document. Example use cases for this API call include sending context or sending ASR blobs to the server.
 
+Events
```
Comment (cbucht200) [inline docs/api/VoiceControlPlugin.md:178]: Same as above

**Review by cbucht200 — APPROVED** [review]
Looks good.

**Review by anand-ky — APPROVED** [review]
